extra-dev/packages/coq-template-coq/coq-template-coq.8.7.dev/descr

```diff
@@ -0,0 +1,17 @@
+A quoting and unquoting library for Coq in Coq.
```

Comment:
This has nothing to do with #208 ...

Reply:
Yeah, I just pushed over another PR, which I'll expect will be merged before that one